microsoft-edge/progressive-web-apps-chromium/experimental-features/index.md

```diff
@@ -179,15 +179,13 @@ For example, the `url_handlers` member value is set to `*.contoso.com` matches `
  
 ## Window Controls Overlay for installed desktop web apps  
 
-To create an immersive title bar similar to a native app for your desktop installed web app.  The **Window Controls Overlay** feature  completes the following actions.  
+To create an immersive title bar similar to a native app for your desktop installed web app, the **Window Controls Overlay** feature  completes the following actions.  
     
 1.  Removes the system reserved title bar.  It usually spans the width of the client frame.  
 1.  Replaces it with an overlay.  It contains just the critical system required window controls necessary for a user to control the window itself.  
     
 After it provides an overlay, the entire web client area is available for you to use.  This feature includes a manifest update.  It provides ways for you to determine the size and position of the overlay to help you arrange content.  
 
-To preview Window Controls Overlay in Microsoft Edge on Windows, navigate to [Turn on experimental features](#turn-on-experimental-features) and turn on **Desktop PWA Window Controls Overlay**. 
```

Comment:
This is the version that I think we should keep since it is associated with the whole feature. Instead, can you remove the similar copy on L265?

Reply:
Yes it's removed.

Reply:
Done.